app-devel/git-filter-repo/autobuild/patch

```diff
@@ -0,0 +1,2 @@
+abinfo "Applying patches ..."
```

Comment:
This file is not necessary, Autobuild applies all patches in autobuild/patches/ sequentially.